Placenote.framework/Headers/libPlacenote.h

```diff
@@ -114,6 +114,10 @@ FOUNDATION_EXPORT int PNSaveMap (const char* mapId, transfer_map_callback cb, vo
 FOUNDATION_EXPORT int PNLoadMap (const char* mapId, transfer_map_callback cb, void* context);
 FOUNDATION_EXPORT int PNDeleteMap (const char* mapId, result_callback cb, void* context);
 FOUNDATION_EXPORT int PNAddMap (result_callback cb, void* context);
+FOUNDATION_EXPORT int PNSyncThumbnail(const char* mapId, const char* thumbnailPath,
```

Comment:
Is this tested with the new version? The last update to the swift library was not tested with the archive that contains the thumbnail functions

Reply:
Yup. Its been tested.